rest_framework/static/rest_framework/docs/js/api.js

```diff
@@ -36,23 +69,23 @@ $('form.api-interaction').submit(function(event) {
     const key = form.data("key");
     var params = {};
 
-    const formData = new FormData(form.get()[0]);
-    for (var [paramKey, paramValue] of formData.entries()) {
+    const entries = formEntries(form.get()[0]);
+    for (var [paramKey, paramValue] of entries) {
```

Comment:
`for..of` support wasn't strong in browsers until very recently: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Browser_compatibility
And assignment destructuring is even more recent: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Browser_compatibility